Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Main {\r\n    public static void main(String[] args) {\r\n        System.out.println(\"Hello World!\");\r\n        System.out.println(\"maneemejeff\");\r\n        System.out.println(\"branch?\");\r\n        //yaaaaaaaaas\r\n        //blablabla\r\n        //gejs\r\n        System.out.println(\"zilais valis\");\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
--- a/src/Main.java	(revision cfc2415f6bf77a6ce61bde07add2044b6aa08df1)
+++ b/src/Main.java	(date 1618493573553)
@@ -1,10 +1,6 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello World!");
-        System.out.println("maneemejeff");
-        System.out.println("branch?");
-        //yaaaaaaaaas
-        //blablabla
+        System.out.println("Sveik pasaul!");
         //gejs
         System.out.println("zilais valis");
     }
